7_drivers_fetch/log/7_nldas_subset_rmk.ind

```diff
@@ -0,0 +1,9 @@
+1_nldas_cubes/log/NLDAS_time[0.4999]_y[132.196]_x[221.344]_var[apcpsfc].nc.ind: bdf7bd70c18464406179d85151e5cc36
+1_nldas_cubes/log/NLDAS_time[0.4999]_y[132.196]_x[221.344]_var[dlwrfsfc].nc.ind: ba1ae3f443eb91b10b710ad016f40dc1
+1_nldas_cubes/log/NLDAS_time[0.4999]_y[132.196]_x[221.344]_var[dswrfsfc].nc.ind: 2120fc5077f72bac272b34927f0afe18
+1_nldas_cubes/log/NLDAS_time[0.4999]_y[132.196]_x[221.344]_var[pressfc].nc.ind: bb461888348a1bfae9d31f0e13fdb59f
```

Comment:
are you just calling `loop_tasks` from the R prompt for now?

Reply:
yes